xml/System/HashCode.xml

```diff
@@ -13,8 +13,53 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Combines the hash code for multiple values into a single hash code.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+You can use <xref:System.HashCode> to combine multiple values (for example, fields on a structure or class) into a
+single hash code. This structure operates in one of two ways:
+
+ * Static methods that accept a set of up to eight values to combine.
+ * Two instance methods that operate in a streaming fashion, accepting values one at a time.
+
+<xref:System.HashCode> uses a static random seed. <xref:System.HashCode> returns stable values across a single process
+invocation, but not across multiple process invocations. Do not store hash codes produced by <xref:System.HashCode> in
+serialized structures, for example, on-disk.
+
+<xref:System.HashCode> currently returns the same value if <xref:System.HashCode.ToHashCode> is called multiple times.
```

Comment:
This paragraph is confusing to me. Without adding the detail you've added in the ToHashCode method, this implies that the same object could generate different hash codes. That sounds bad.

Going further, it implies that if a type supports value-based equality, two objects with the values (and therefore considered equal) may not always generate the same hash code. That also sounds bad.

I'm probably missing some important point, and that important point should be made more clear.

Reply:
I was worried when I read it that this test might fail:

```csharp
var hc1 = new HashCode();
hc1.Add(1);
hc1.Add(2);
var hashOne = hc1.ToHashCode();
var hc2 = new HashCode();
hc2.Add(1);
hc2.Add(2);
var hashTwo = hc2.ToHashCode();
Assert.Equals(hashOne, hashTwo);
```

I think I understand, but there's a bug in your test above (`b` isn't declared).  If you make a correction, I'll suggest different wording.

Reply:
Yep, I meant to combine `(1, 2)`. I've submitted `Dispose`-inspired wording that simply omits the explanation as to why.